components/Document/pages/DocumentPageLayout.tsx

```diff
@@ -19,12 +18,15 @@ import { useState } from "react";
 
 interface Args {
   document: GenericDocument;
-  errorCode?: number;
-  documentType: DocumentType;
-  tabName?: string;
   children?: any;
   metadata: DocumentMetadata;
   isExpanded?: boolean;
+  noLineItems?: boolean;
```

Comment:
Related to my comment below, I think having a minimal header will avoid us having to configure this component as well